src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, setSelectedTabs, state, setState,
+  onTabSelected }) => (
```

Comment:
use more meaningful names, it is difficult to understand what `state` is and what it means